drainer/syncer.go

```diff
@@ -660,7 +660,9 @@ func skipDMLEvent(pv *pb.PrewriteValue, schema *Schema, filter *filter.Filter, b
 			mutation.DeletedRows = mutation.DeletedRows[0:filteredDeleteIdx]
 		}
 
-		muts = append(muts, mutation)
+		if len(mutation.Sequence) > 0 {
```

Comment:
if it's filtered out, `len(mutation.Sequence) == 0` ?

Reply:
it's in line 657, where `filteredIdx` never get `++`